system-parachains/coretime/coretime-polkadot/src/coretime.rs

```diff
@@ -335,5 +347,9 @@ impl pallet_broker::Config for Runtime {
 	type WeightInfo = weights::pallet_broker::WeightInfo<Runtime>;
 	type PalletId = BrokerPalletId;
 	type AdminOrigin = EnsureRoot<AccountId>;
+	type SovereignAccountOf = SovereignAccountOf;
+	// Roughly 1.5 years with a period of 28 days(~ish)
+	// TODO: @szegoo, can you help checking this value?
+	type MaxAutoRenewals = ConstU32<20>;
```

Comment:
I think 100 would be a reasonable value for the short- to mid-term future. As the number of cores Polkadot offers increases, this may be subject to change. However, we should also keep in mind that not all core owners will enable auto-renewal.

```suggestion
	type MaxAutoRenewals = ConstU32<100>;
```

Lets also confirm with @seadanda if this sounds good.

Reply:
100 sounds reasonable. The aim here is not to create competition for the autorenew slots but just to set a bound on the storage and this will cover it for the next couple of iterations. long term the aim is for 200 cores on Polkadot, but we're not there yet and can update this when the time comes. Like you mentioned not all core owners will enable it, but we still don't have an idea of the percentage of cores which will be renewed.